flavors/posture.go

```diff
@@ -74,9 +74,9 @@ func NewPosture(_ *beat.Beat, cfg *agentconfig.C) (*posture, error) {
 		return nil, err
 	}
 
-	// TODO: timeout should be configurable and not hard-coded. Setting to 5 minutes for now to account for CSPM fetchers
+	// TODO: timeout should be configurable and not hard-coded. Setting to 10 minutes for now to account for CSPM fetchers
 	// 	https://github.com/elastic/cloudbeat/issues/653
-	data, err := fetchersManager.NewData(log, c.Period, time.Minute*5, fetchersRegistry)
+	data, err := fetchersManager.NewData(log, c.Period, time.Minute*10, fetchersRegistry)
```

Comment:
Honestly, having it configurable is the best approach. First, who should configure it, the user? from the integration page? how will the user know what is the right timeout? what is the other option? users will see an issue and will contact us? again, not the best solution since we will also not know what is the correct timeout, and most users will not even notice that there is an issue. I think the right approach should be something robust that will not require configuration from the user at all.

Reply:
Got it, in my opinion, it should not be configurable by the user and this issue should be handled programmatically.
I'll open a ticket to discuss the matter.

Reply:
There's an issue in the comment https://github.com/elastic/cloudbeat/issues/653 you can expand the discussion